File: 07_03_2022/Toggle_Switch.py
n = int(input())
m = int(input())
x = False
for i in range(1,m+1):
    if(m%i == 0):
        if(x == False):
            x = True
        else:
            x = False

if(x == False):
    print("Off")
else:
    print("On")

File: 07_03_2022/Toggle_Switch.cpp
#include <iostream>

using namespace std;

int main(){

    int n,m;
    cin>>n>>m;

    bool x = false;
    for(int i=1;i<m+1;i++){
        if(m%i == 0){
            if(x == false) x = true;
            else x = false;
        }
    }

    if(x == false){
        cout<<"Off";
    }
    else{
        cout<<"On";
    }

    return 0;
}